### PR TITLE
Fix edge-to-edge for the full-screen player

### PR DESCRIPTION
**Pull request by geekygecko**
## Description

This change focuses on converting the full-screen to edge-to-edge for the Android 15 project. Please ignore the other pages for now. 

There will be multiple pull requests required to support Android 15 edge-to-edge but I'm going to break them down into smaller PRs. I will merge them all into a `task/android-15` branch, and after all the work is completed, raise a PR to merge into `main`.

## Testing Instructions

Open the following pages and verify the content isn't under the status or navigation bar.

1. Subscribe to Stuff You Should Know (a podcast with transcripts)
2. Open the full-screen page
3. Tap on the effects icon
4. Close and tap on the sleep icon
5. Close and tap on the rearrange three dots
6. Tap on the pencil to edit the actions
7. Go back to the full-screen player
8. Tap on the Up Next icon
9. Go back and open the transcripts page

### Now Playing tab

**Before**

Screenshot 2024-12-05 at 2 14 20 pm

**After**

Screenshot 2024-12-05 at 1 38 50 pm

### Up Next

**Before**

Screenshot 2024-12-05 at 2 15 41 pm

**After**

Screenshot 2024-12-05 at 1 40 31 pm

### Effects 

**Before**

Screenshot 2024-12-05 at 2 17 50 pm

**After**

The navigation bar on SDK 35 isn't quite right for the bottom sheet but I'm going to leave this for later.

Screenshot 2024-12-05 at 10 17 00 pm

### Sleep 

**Before**

Screenshot 2024-12-05 at 2 18 02 pm

**After**

Screenshot 2024-12-05 at 10 17 14 pm

### More actions 

**Before**

Screenshot 2024-12-05 at 2 20 32 pm

**After**

Screenshot 2024-12-05 at 10 19 07 pm

### Rearrange actions

**Before**

Screenshot 2024-12-05 at 2 19 34 pm

**After**

Screenshot 2024-12-05 at 3 41 41 pm

### Transcripts

**Before**

Screenshot 2024-12-05 at 3 56 11 pm

**After**

Screenshot 2024-12-05 at 3 51 24 pm

## Checklist
- [x] If this is a user-facing change, I have added an entry in CHANGELOG.md
- [x] Ensure the linter passes (`./gradlew spotlessApply` to automatically apply formatting/linting)
- [x] I have considered whether it makes sense to add tests for my changes
- [x] All strings that need to be localized are in `modules/services/localization/src/main/res/values/strings.xml`
- [x] Any jetpack compose components I added or changed are covered by compose previews
- [x] I have updated (or requested that someone edit) [the spreadsheet](https://docs.google.com/spreadsheets/d/107jqrutZhU0fVZJ19SBqxxVKbV2NWSdQC9MFYdLiAxc/edit?usp=sharing) to reflect any new or changed analytics.
 
#### I have tested any UI changes...

- [x] with different themes
- [x] with a landscape orientation
- [x] with the device set to have a large display and font size
- [x] for accessibility with TalkBack

### CODE REVIEW

**Comment by dangermattic** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          1 Message
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">This PR is still a Draft: some checks will be skipped.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr-check">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: modules/features/player/src/main/java/au/com/shiftyjelly/pocketcasts/player/view/ShelfFragment.kt [hunk 1]
```diff
@@ -41,7 +42,7 @@ class ShelfFragment : BaseFragment() {
         container: ViewGroup?,
         savedInstanceState: Bundle?,
     ) = content {
-        AppTheme(theme.activeTheme) {
+        AppTheme(Theme.ThemeType.DARK) {
```
Comment (geekygecko) [inline modules/features/player/src/main/java/au/com/shiftyjelly/pocketcasts/player/view/ShelfFragment.kt:45]: The player theme is always dark.

**Comment by dangermattic** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">This PR is larger than 500 lines of changes. Please consider splitting it into smaller PRs for easier and faster reviews.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr-check">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Review by mebarbosa — APPROVED** [review]
The code looks good and everything worked well in my tests 💯

**Pull request merged**